Due to automake stupidity the Makefile in tests/ has an 'all' target that
depends on $(BUILD_SOURCES), which means when building flex parts of the test
suite are built even if they're not needed, resulting in flex needing
flex-native and bison-native to build.

This patch removes the tests directory from SUBDIRS and will be conditionally
applied by the recipe.

Upstream-Status: Inappropriate
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/Makefile.am b/Makefile.am
index 076ccad..0574d7b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -55,3 +55,2 @@ SUBDIRS = \
 	po \
-	tests \
 	tools
